packages/basti-cdk/package.json

```diff
@@ -46,8 +48,19 @@
     "jsii-pacmak": "^1.85.0",
     "ts-jest": "^29.1.1"
   },
-  "peerDependencies": {
-    "aws-cdk-lib": "2.86.0",
-    "constructs": "10.0.5"
+  "jsii": {
+    "outdir": "dist/packages",
+    "versionFormat": "full",
+    "targets": {
+      "python": {
+        "distName": "basti-cdk",
+        "module": "basti_cdk"
+      }
+    },
+    "tsc": {
```

Comment:
This is how `jsii` provides some control over the generated `tsconfig.json` file.